app/experimenter/nimbus-ui/src/components/PageResults/TableHighlights/index.tsx

```diff
@@ -65,25 +72,34 @@ const TableHighlights = ({
                 <span className="align-middle">All Users&nbsp;</span>
                 <Info data-tip={SEGMENT_TIPS.ALL_USERS} />
               </td>
-              <td className="pt-3 px-3 pb-0">
-                {highlightMetricsList.map((metric) => {
-                  const metricKey = metric.value;
-                  const displayType = getTableDisplayType(
-                    metricKey,
-                    TABLE_LABEL.HIGHLIGHTS,
-                    results[branch]["is_control"],
-                  );
-                  return (
-                    <TableVisualizationRow
-                      key={`${displayType}-${metricKey}`}
-                      metricName={metric.name}
-                      results={results[branch]}
-                      tableLabel={TABLE_LABEL.HIGHLIGHTS}
-                      {...{ metricKey, displayType }}
-                    />
-                  );
-                })}
-              </td>
+              {overallResults[branch]["is_control"] ? (
+                <td className="p-1 p-lg-3 align-middle">
+                  <div className="font-italic align-middle">---baseline---</div>
+                </td>
+              ) : (
+                <td className="pt-3 px-3">
+                  {highlightMetricsList.map((metric) => {
+                    const metricKey = metric.value;
+                    const displayType = getTableDisplayType(
+                      metricKey,
+                      TABLE_LABEL.HIGHLIGHTS,
+                      overallResults[branch]["is_control"],
+                    );
+                    const tooltip =
+                      results.metadata?.metrics[metricKey]?.description ||
```

Comment:
Huh, I'll look more closely at this tomorrow morning but I'm not seeing this come through on Storybook. 🤔

Reply:
Where were you checking on storybook? The screenshot I shared is from storybook so I believe it should show up there.

Reply:
I totally see it now! I was looking at search increased, not decreased.

<img width="454" alt="image" src="https://user-images.githubusercontent.com/13018240/108871817-267f1c80-75bf-11eb-913e-35f601ad7323.png">

BTW, I noticed that some of the tooltips are not at total opacity and others are. It seems like especially the one with the blue link should be on the darker side? This is a tiny nit though (could file a follow up as a `quick-win` to update the opacity for consistency if you'd prefer to get this in and avoid another CI run). I also wonder, do users know that they can see all this nice extra info by hovering over certain pieces?

<img width="462" alt="image" src="https://user-images.githubusercontent.com/13018240/108871892-3ac31980-75bf-11eb-916d-1d923dafd6a1.png">

Reply:
I'll file a follow up about the opacity. In user interviews we've seen people try to hover over everything for more info. A little "?" icon might be a bit better but I think we can wait to see if we get that as direct feedback from upcoming interviews

Reply:
Interesting. Is the difference in opacity expected? Or is it possible that multiple tooltips are being rendered/layering to create an opaque effect?